configure.ac

```diff
@@ -2677,6 +2677,19 @@ AC_CHECK_LIB(sendfile, sendfile)
 AC_CHECK_LIB(dl, dlopen)	# Dynamic linking for SunOS/Solaris and SYSV
 AC_CHECK_LIB(dld, shl_load)	# Dynamic linking for HP-UX
 
+AC_CACHE_CHECK([if uuid_generate_time_safe() available],
+[ax_cv_uuid_generate_time_safe],
+[AC_RUN_IFELSE([
+    AC_LANG_PROGRAM([#include <uuid/uuih.h>], [
+        uuid_t out;
+        uuid_generate_time_safe(out);
+        ])],
+    [ax_cv_uuid_generate_time_safe=yes],
```

Comment:
Is it possible to put directly the AC_DEFINE() here, instead of using a variable?

[ax_cv_uuid_generate_time_safe=no] would be replaced with [] in that case.

Reply:
I tried that but I got the following warning:

```
configure.ac:2688: warning: AC_CACHE_VAL(ax_cv_uuid_generate_time_safe, ...): suspicious presence of an AC_DEFINE in the second argument, where no actions should be taken
```

Reply:
Oh, I hate autotools, mysterious tools... Ok, leave it as it is ;-)